incubator/monochart/templates/daemonset.yaml

```diff
@@ -0,0 +1,107 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: {{ include "common.fullname" . }}
+  annotations:
+{{ toYaml .Values.daemonset.annotations | indent 4 }}
+  labels:
+{{ include "common.labels.standard" . | indent 4 }}
+spec:
+  template:
+    metadata:
+      name: {{ include "common.fullname" . }}
+      annotations:
+{{ toYaml .Values.daemonset.pod.annotations | indent 8 }}
+      labels:
+        app: {{ template "common.name" . }}
+        release: {{ .Release.Name | quote }}
+    spec:
+      containers:
+      - name: {{ .Chart.Name }}
+        image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        imagePullSecrets:
+{{ toYaml .Values.image.pullSecrets | indent 10 }}
+        envFrom:
+        - configMapRef:
+            name: {{ include "common.fullname" . }}
+        env:
+{{- range $name, $value := .Values.env.open }}
```

Comment:
We need to pick one of these ways of exporting envs, because currently we export all envs twice. 

Maybe we use this values format?
```
env:
  configMap:
  secrets:
  inline
```

Reply:
Or maybe we should use this format:

```
configMap:
  env:
    USER: bar
  files:
     "/etc/test.conf": "foo"

secrets:
  env:
    PASS: secret
  files:
     "/etc/ssl/test.crt": |-
       ...

# Inline env
env:
  USER: test
```